tests/test_torch_dftd3_calculator.py

```diff
@@ -105,25 +124,22 @@ def _test_calc_energy_force_stress(
             abc=abc,
             bidirectional=bidirectional,
         )
-        _assert_energy_force_stress_equal(dftd3_calc, torch_dftd3_calc, atoms)
+        _assert_energy_force_stress_equal(dftd3_calc, torch_dftd3_calc, atoms, force_tol=force_tol)
```

Comment:
I found that FP32 sometimes fail on accuracy check. I modified the threshold for FP32 cases.